node/src/protocol/consensus.rs

```diff
@@ -310,6 +322,12 @@ impl ConsensusProtocol for WaitingForConsensusState {
                     }
                     let participants_vec: Vec<Participant> =
                         self.participants.keys().cloned().collect();
+
+                    let me = contract_state
+                        .participants
+                        .find_participant(ctx.my_account_id())
+                        .unwrap();
```

Comment:
This is going to be a brittle pattern going forward. `me()` does not exist in all contexts so we should strive to make this type safe and ensure we can only use it in the contexts where it exists. One potential solution to this is https://github.com/near/mpc-recovery/issues/416